src/current/v24.1/create-schedule-for-backup.md

```diff
@@ -10,7 +10,7 @@ docs_area: reference.sql
 For more information about creating, managing, monitoring, and restoring from a scheduled backup, see [Manage a Backup Schedule]({% link {{ page.version.version }}/manage-a-backup-schedule.md %}).
 
 {{site.data.alerts.callout_info}}
-Core users can only use backup scheduling for [full backups](#create-a-schedule-for-full-backups-only-core) of clusters, databases, or tables. If you do not specify the `FULL BACKUP ALWAYS` clause when you schedule a backup, you will receive a warning that the schedule will only run full backups. 
+Core users can only use backup scheduling for [full backups](#create-a-schedule-for-full-backups-only-core) of clusters, databases, or tables. If you do not specify the `FULL BACKUP ALWAYS` clause when you schedule a backup, you will receive a warning that the schedule will only run full backups.
```

Comment:
total nit but it would be easier to review a big PR like this without formatting changes and leave those for a separate quick PR

Reply:
This was just my editor removing spaces where I hadn't had it enabled when previously working on this page. Apologies

Reply:
ah no biggie! auto editor trailing whitespace removal is a good thing, I should set that up